### ISSUE TITLE
Exercise 2

### ISSUE THREAD

**Issue by vatsalya1103** [nus-cs2030/2223-s2#52]
How to compare the type of objects without using instanceof as codecrunch throws an error.

**Comment by trwstin** [comment]
I don't think comparison between objects is necessary for this exercise. If you cannot find a workaround for instanceof, perhaps you could try a different method?

**Comment by trwstin** [comment]
I don't think it is necessary to compare objects for this exercise. If you are having trouble finding a workaround for instanceof, perhaps you can try thinking of an alternate method? @e0959170's suggestion is a good start.

**Comment by AmanDesai34** [comment]
Yea, i think @e0959170's method sounds good! Do let us know if it worked, else we can try  to think of another method
